### ISSUE TITLE
Failed to load extension list

### ISSUE THREAD

**Issue by lynbor** [smiffy6969/razorCMS#47]
Working on my XAMPP setup under Win7 this works just fine, but after uploading to my production hosting account (VPS cPanel/WHM at JaguarPC) I get the error "Failed to load extension list".  To get around the problem I use my XAMPP installation to install the extension I want and then use FTP to copy the files up to my production server.  At least that has worked for different themes I wanted to use.  

I have not tried anything more than a couple themes.

**Comment by smiffy6969** [comment]
Has your server got PHP curl module loaded?

via CloudMagic Email [https://cloudmagic.com/k/d/mailapp?ct=pa&cv=8.5.49&pv=6.0.1&source=email_footer_2] On Wed, Jul 13, 2016 at 4:21 PM, lynbor < notifications@github.com [notifications@github.com] > wrote:
Working on my XAMPP setup under Win7 this works just fine, but after uploading to my production hosting account (VPS cPanel/WHM at JaguarPC) I get the error "Failed to load extension list". To get around the problem I use my XAMPP installation to install the extension I want and then use FTP to copy the files up to my production server. At least that has worked for different themes I wanted to use.

I have not tried anything more than a couple themes.

**Comment by lynbor** [comment]
Yes it is..  put this code in a file in the same folder as razorcms and ran it.
```
<?php

// Script to test if the CURL extension is installed on this server

// Define function to test
function _is_curl_installed() {
    if  (in_array  ('curl', get_loaded_extensions())) {
        return true;
    }
    else {
        return false;
    }
}

// Ouput text to user based on test
if (_is_curl_installed()) {
  echo "cURL is <span style=\"color:blue\">installed</span> on this server";
} else {
  echo "cURL is NOT <span style=\"color:red\">installed</span> on this server";
}
?>
```
returns...

cURL is installed on this server
